connectors/connectors_cli.py

```diff
@@ -177,6 +199,35 @@ def prompt():
             hide_input=True if item.get("sensitive") is True else False,
         )
 
+    index_exists, connector_exists = index.index_or_connector_exists(index_name)
+
+    if index_exists and not use_existing_index:
+        click.echo(
+            click.style(
+                f"Index for {index_name} already exists. Include the flag `--use_existing_index` to create a connector for this index.",
+                fg="red",
+            ),
+            err=True,
+        )
+        return
```

Comment:
What is the status of the command in this case?

Reply:
Should I raise the error instead? I didn't think to ensure the exit code isn't `0`.
Or maybe this is better: https://click.palletsprojects.com/en/8.0.x/api/#click.Abort

Reply:
Added `raise click.Abort()` to the index/connector exists catches